CRM/Upgrade/Incremental/sql/4.7.32.mysql.tpl

```diff
@@ -1 +1,6 @@
 {* file to handle db changes in 4.7.32 during upgrade *}
+
+-- CRM-21827: Add Membership External Id as a new field --
```

Comment:
would recommend altering this to be in the PHP file `CRM/Upgrade/Incremental/php/FourSeven.php` like https://github.com/civicrm/civicrm-core/blob/master/CRM/Upgrade/Incremental/php/FourSeven.php#L505